src/models/pats.rs

```diff
@@ -132,7 +132,10 @@ impl Scopes {
     }
 
     pub fn parse_from_oauth_scopes(scopes: &str) -> Result<Scopes, bitflags::parser::ParseError> {
-        let scopes = scopes.replace(' ', "|").replace("%20", "|");
+        let scopes = scopes
```

Comment:
Clippy issue here- make sure you run `cargo clippy` (and maybe `--fix`) :)

Reply:
Man clippy and rust in general is so cool